luigi_pipeline/lib/hail_tasks.py

```diff
@@ -18,22 +18,86 @@
 
 logger = logging.getLogger(__name__)
 
+GRCh37_STANDARD_CONTIGS = {
+    "1",
+    "10",
+    "11",
+    "12",
+    "13",
+    "14",
+    "15",
+    "16",
+    "17",
+    "18",
+    "19",
+    "2",
+    "20",
+    "21",
+    "22",
+    "3",
+    "4",
+    "5",
+    "6",
+    "7",
+    "8",
+    "9",
+    "X",
+    "Y",
+    "MT",
+}
+GRCh38_STANDARD_CONTIGS = {
+    "chr1",
+    "chr10",
+    "chr11",
+    "chr12",
+    "chr13",
+    "chr14",
+    "chr15",
+    "chr16",
+    "chr17",
+    "chr18",
+    "chr19",
+    "chr2",
+    "chr20",
+    "chr21",
+    "chr22",
+    "chr3",
+    "chr4",
+    "chr5",
+    "chr6",
+    "chr7",
+    "chr8",
+    "chr9",
+    "chrX",
+    "chrY",
+    "chrM",
+}
+OPTIONAL_CHROMOSOMES = ["MT", "chrM", "Y", "chrY"]
+VARIANT_THRESHOLD = 100
+CONST_GRCh37 = "37"
+CONST_GRCh38 = "38"
+
 
 class MatrixTableSampleSetError(Exception):
     def __init__(self, message, missing_samples):
         super().__init__(message)
         self.missing_samples = missing_samples
 
 
+class VCFValidationError(Exception):
+    pass
+
+
 def GCSorLocalTarget(filename):
-    target = gcs.GCSTarget if filename.startswith('gs://') else luigi.LocalTarget
+    target = gcs.GCSTarget if filename.startswith("gs://") else luigi.LocalTarget
```

Comment:
Looks like your linter may be set to double quotes 🤣

Reply:
dangit